pkg/kubehound/graph/edge/role_bind_rb_rb_r.go

```diff
@@ -127,12 +130,48 @@ func (e *RoleBindRbRbR) Stream(ctx context.Context, store storedb.Provider, c ca
 		// Looking for all permissionSets link to the same namespace
 		bson.M{
 			"$lookup": bson.M{
-				"from":         "permissionsets",
-				"localField":   "namespace",
-				"foreignField": "namespace",
-				"as":           "linkpermset",
+				"as":   "linkpermset",
+				"from": "permissionsets",
+				"let": bson.M{
+					"roleNamespace": "$namespace",
+				},
+				"pipeline": []bson.M{
+					{
+						"$match": bson.M{
+							"$or": bson.A{
+								bson.M{"$expr": bson.M{
+									"$eq": bson.A{
+										"$k8.objectmeta.namespace", "$$roleNamespace",
+									},
+								}},
+								bson.M{"is_namespaced": true},
+							},
+							"runtime.runID":   runtime.RunID.String(),
+							"runtime.cluster": runtime.ClusterName,
+						},
+					},
+					{
+						"$project": bson.M{
+							"_id": 1,
+						},
+					},
+				},
 			},
 		},
+		// Looking for all permissionSets link to the same namespace
```

Comment:
Does we want to keep that comment? Why?

Reply:
No removing it. This was a first try using a simple version, but did not work properly.